docs/source/getting_started/quickstart.rst

```diff
@@ -40,6 +40,14 @@ Initialize vLLM's engine for offline inference with the ``LLM`` class and the `O
 
     llm = LLM(model="facebook/opt-125m")
 
+Use model from www.modelscope.cn
+.. code-block:: shell
+    export VLLM_USE_MODELSCOPE=True
+
+.. code-block:: python
+
+    llm = LLM(model="damo/nlp_gpt2_text-generation_english-base", revision="v1.0.0")
```

Comment:
I think "qwen/Qwen-7B-Chat" may be a better option to be used as an example.

Reply:
updated